<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export 3D Volume</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QListWidget" name="listsel_layer"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_settings_message">
     <property name="text">
      <string>Set the export settings for each layer</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_filetype_message">
     <property name="text">
      <string>Select the export filetype</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="combosel_filetype"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_compression_message">
     <property name="text">
      <string>Select compression method</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>110</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_ok">
        <property name="text">
         <string>Export</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <layout class="QFormLayout" name="layer_layout"/>
   </item>
   <item row="8" column="0">
    <widget class="QComboBox" name="combosel_compression"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
